upgrade/sql/8.0.0.sql

```diff
@@ -26,7 +26,8 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'displayCheckoutBeforeConfirmation', 'Show custom content before checkout confirmation', 'This hook allows you to display custom content at the end of checkout process', '1'),
   (NULL, 'displayCheckoutSummaryTop', 'Cart summary top', 'This hook allows you to display new elements in top of cart summary', '1'),
   (NULL, 'displayAdminThemesListAfter', 'BO themes list extra content', 'This hook displays content after the themes list in the back office', '1'),
-  (NULL, 'displayModuleConfigureExtraButtons', 'Module configuration - After toolbar buttons', 'This hook allows to add toolbar''s additional content on module configuration page', '1')
+  (NULL, 'displayModuleConfigureExtraButtons', 'Module configuration - After toolbar buttons', 'This hook allows to add toolbar''s additional content on module configuration page', '1'),
+  (NULL, 'actionGetModulesExtraDescription', 'Module extra description', 'This hook allows to add an extra description to display in the Module manager page', '1')
```

Comment:
Hi @sowbiba 

I don't understand I can't find the hook `actionGetModulesExtraDescription` anywhere in the core, and even the PR you linked to this issue https://github.com/PrestaShop/PrestaShop/pull/28330 doesn't contain any new hook.

Are you sure this PR is really needed? Or is it that something is missing in the initial PR on the core?